### PR TITLE
add option for different executablePath for the scraper

### PR DESCRIPTION
**Pull request by shaiu**

### CODE REVIEW

**Comment by daniel-hauser** [comment]
Can you share more about the need? 

From [israeli-bank-scrapers](https://github.com/eshaham/israeli-bank-scrapers/blob/80c67579b91a329184417dea9ecb8be5bb8fccdc/src/scrapers/interface.ts#L61:L65):
> `executablePath` provide a patch to local chromium to be used by puppeteer. Relevant when using `israeli-bank-scrapers-core` library

**Comment by shaiu** [comment]
@daniel-hauser It's related to https://github.com/daniel-hauser/moneyman/issues/265

When trying to work locally on my M1 Mac I need to install the chromium manually.

Currently with moneyman code the path to chromium is hard coded and this PR gives the option to point to a different path

**Comment by daniel-hauser** [comment]
Can you add the new `PUPPETEER_EXECUTABLE_PATH` setting to the table in the readme?

**Comment by shaiu** [comment]
> Can you add the new `PUPPETEER_EXECUTABLE_PATH` setting to the table in the readme?

Added

File: README.md [hunk 1]
```diff
@@ -89,13 +89,14 @@ Example:
...
 | `DAYS_BACK`             | `10`               | The amount of days back to scrape                                                                                                             |
 | `TZ`                    | `'Asia/Jerusalem'` | A timezone for the process - used for the formatting of the timestamp                                                                         |
 | `FUTURE_MONTHS`         | `1`                | The amount of months that will be scrapped in the future, starting from the day calculated using `DAYS_BACK`                                  |
 | `TRANSACTION_HASH_TYPE` | ``                 | The hash type to use for the transaction hash. Can be `moneyman` or empty. The default will be changed to `moneyman` in the upcoming versions |
 | `HIDDEN_DEPRECATIONS`   | ''                 | A comma separated list of deprecations to hide                                                                                                |
+| `PUPPETEER_EXECUTABLE_PATH`   | undefined          | An ExecutablePath for the scraper. if undefined defaults to system.                                                                           |
```
Comment (daniel-hauser) [inline README.md]:
```suggestion
| `PUPPETEER_EXECUTABLE_PATH`   | `undefined`          | An ExecutablePath for the scraper. if undefined defaults to system.                                                                           |
```

**Pull request merged**